recipes/gif/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "gif" %}
+{% set version = "22.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gif-{{ version }}.tar.gz
+  sha256: f4efc868c44d8738f2c6342918266634d5202a9fe19f761f373869109f541bdc
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
```

Comment:
```suggestion
    - pip
    - setuptools >=62.1.0
```

Reply:
Not sure how much we actually care since it worked, but that's listed under `setup_requires`.

Reply:
>= usually passes b/c that is what conda solve will find. But better be explicit than dealing with weird errors later.